package.json

```diff
@@ -2,18 +2,22 @@
   "name": "xterm-pty",
   "description": "A pty for xterm.js and Emscripten",
   "version": "0.10.2",
-  "main": "index.js",
+  "main": "out/index.js",
```

Comment:
Do you need to change this? I would like `index.js` to be one minified by webpack.